src/me/StevenLawson/TotalFreedomMod/Commands/Command_blockcmd.java

```diff
@@ -8,7 +8,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
-@CommandParameters(description = "Block all commands for a specific player.", usage = "/<command> <purge | <partialname>>", aliases = "blockcommands,blockcommand")
+@CommandParameters(description = "Block all commands for a specific player.", usage = "/<command> <purge | <partialname> [-a]>", aliases = "blockcommands,blockcommand")
```

Comment:
`purge` and `partialname` become optional now that `-a` is added. Please make this is a three-way OR clause.